lua/nvim-web-devicons.lua

```diff
@@ -1736,7 +1736,7 @@ local function get_default_icon()
 end
 
 local function get_icon(name, ext, opts)
-  ext = ext or name:match("^.*%.(.*)$") or ""
+  ext = ext or name:match("^.*%.(.*%..*)$") or name:match("^.*%.(.*)$") or ""
```

Comment:
Could we simply match `"^.*%.(.*%..*)$"` without `"^.*%.(.*)$"`? Greedy match should cover all existing cases...

Reply:
@alex-courtis Thanks for your feedback. I've updated the regexp pattern to match both 'foo.test.js' and 'foo.js'. Could you double check?